CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 
 ## Prerelease
 
+### Adds
+
+- Adds the `labelFontSize` prop to the `StatusBadge` component.
```

Comment:
This should be updated to reflect that internal update for the font size but no new props were added.